### ISSUE TITLE
LinkedIn can't be scraped?

### ISSUE THREAD

**Issue by cwills** [jaimeiniesta/metainspector#209]
Anyone find a way to scrape LinkedIn URLs?

Example: https://www.linkedin.com/pulse/artificial-intelligence-ai-new-sales-frontier-timothy-tim-hughes

**Comment by jaimeiniesta** [comment]
I can scrape it from my development machine just fine using MetaInspector 5.4.1:

```ruby
p = MetaInspector.new("https://www.linkedin.com/pulse/artificial-intelligence-ai-new-sales-frontier-timothy-tim-hughes")
p.title
# "Is Artificial Intelligence (AI) the New Sales Frontier?\n | LinkedIn\n "
```

But it fails from the [demo hosted in Heroku](https://metainspectordemo.herokuapp.com/scrape?url=https%3A%2F%2Fwww.linkedin.com%2Fpulse%2Fartificial-intelligence-ai-new-sales-frontier-timothy-tim-hughes)

From the output I see a 999 response status code, that seems [the way that LinkedIn has for a request denied response](https://www.google.es/search?q=999+response+status).

So in short, LinkedIn might be blocking your request based on your server IP. There's not much you can do in this case I'm afraid, as the base request will be blocked. You could try setting a different User-Agent string (see README.md) but it looks like it's your IP being blocked.

**Comment by cwills** [comment]
Thank you. I'm getting the 999 response status code on both my Cloud9 development account and on my production account.

**Comment by jaimeiniesta** [comment]
Makes sense, it could be that both Cloud9 and your production servers IP are not trusted by LinkedIn. I had the same problem with Heroku - but works fine in my local computer.

**Comment by jaimeiniesta** [comment]
Closing this, feel free to reopen if you think it's not related with IP blocking.

**Comment by malav1410** [comment]
Is anyone found solution for linkedin Parse?

**Comment by jaimeiniesta** [comment]
There's nothing we can do here, LinkedIn is blocking the request based on the IP, that's why it can work on your dev machine but fail when you do it from your production server.

The only thing you can try is provide a different User-Agent and see if Linkedin likes it. If it's based on IP, there's nothing you can do - except use some kind of proxy cloud to fake your IP.

https://stackoverflow.com/questions/27231113/999-error-code-on-head-request-to-linkedin

**Comment by malav1410** [comment]
@jaimeiniesta Thanks.
